Topics/Software_Engineering/Retro.md

```diff
@@ -0,0 +1,88 @@
+# Retro
+
+## Table of Contents
+#### [Introduction](#introduction-1)
+#### [What should a retro encompass?](#what-should-a-retro-encompass-1)
+#### [Is a Retro Meeting Neccessary?](#is-a-retro-meeting-neccessary-1)
+#### [Planning a Retro Meeting](#planning-a-retro-meeting-1)
+#### [Key Takeaways](#key-takeaways-1)
+#### [Resources](#resources-1)
+
+## Introduction
+
+A Retro is a meeting that is usually reserved for the end of a sprint, and goes about reviewing the work completion, overall team engagement, and bring up success/failure stories that occurred during the sprint. The purpose of reviewing this as a team is to gauge the general work ethic of the team, as well as provide a space to bring up issues that would otherwise fall to the wayside during other meetings. Ultimately, by runnning a retro, it provides an opportunity to re-evaluate what happened during the sprint, and to help induce higher-level problem solving to address the potential issues brought up.
+
+## What should a retro encompass?
+
+A Retro typically encompasses three primary features that help build towards a proactive meeting. As a product manager or person running this sprint, try to answer these questions when applicable.
+
+1. Sprint Overview
+- How many tasks were completed during the sprint? Their total points? (if applicable)
+
+2. Sprint Impression
+- How did the team feel about this sprint?
+- If they can describe the sprint in one word, what would it be?
+- What went well? What went wrong?
+- What sprint goals were achieved?
+- What roadblocks occurred?
+- Were there any interpersonal conflicts?
+- Who was unavailable during the sprint?
+
+## Is a Retro Meeting Neccessary?
+
+Retro Meetings tend to get overlooked due teams tending to merge Sprint Review meetings together with Retro Meetings to save time. This isn't a bad thing, but neglecting to put the time and effort to compile and summarize the overall team impressions, successes, and failures are. In short, you can try to condense the meeting, but make sure you provide the space to bring up/address any issues.
+
+If you or your team is able to properly find a consistent schedule to review the sprint and keep some form of documentation, and your team is confident they can problem solve these issues during the sprint, perhaps you may forgo a meeting, and have a living document instead. However, you should err away from this solution if you think your team does not communicate rapidly and consistently over the course of a sprint.
+
+Finding time to review successes, concerns, and failures gives the team a chance to be aware of potential issues as a group, and gives the team an opportunity to problem-solve in a group environment, giving everyone a chance to speak their mind.
+
+## Planning a Retro Meeting
```

Comment:
You could add more guidance for the remote team if you like. This can include how to conduct retrospectives effectively in a remote setting. Moreover, you could cover more general aspects like ensuring engagement in a virtual environment, managing different time zones, and using digital tools effectively.